tests/framework/installer/ceph_installer.go

```diff
@@ -458,7 +458,7 @@ func NewCephInstaller(t func() *testing.T, clientset *kubernetes.Clientset, rook
 		helmHelper:      utils.NewHelmHelper(Env.Helm),
 		k8sVersion:      version.String(),
 		cephVersion:     cephVersion,
-		changeHostnames: rookVersion != Version0_8 && k8shelp.VersionAtLeast("v1.11.0"),
+		changeHostnames: rookVersion != Version0_8 && k8shelp.VersionAtLeast("v1.13.0"),
```

Comment:
help me understand why this changed as well

Reply:
Checking for a k8s version in this case is only to give us test coverage. There isn't a specific dependency on a version for changing the host name. I only changed it to make sure we don't miss updating it when we drop supporting 1.11 at some point down the road.